pkgs/applications/office/libreoffice/default.nix

```diff
@@ -168,7 +168,7 @@ in stdenv.mkDerivation rec {
       ln -s $out/lib/libreoffice/program/$a $out/bin/$a
       wrapProgram "$out/bin/$a" \
          --prefix XDG_DATA_DIRS : \
-         "$out/share:$GSETTINGS_SCHEMAS_PATH" \
+         "$out/share" \
```

Comment:
This is also part of `wrapGAppsHook`: https://github.com/NixOS/nixpkgs/blob/73746502d80878780a771a3d8d56fed071016ca4/pkgs/build-support/setup-hooks/wrap-gapps-hook.sh#L29-L31